### DISCUSSION TITLE
How to correctly backup and restore ~/.local/share/containers?

### DISCUSSION THREAD

**Discussion by space88man** [containers/podman#20487] [General]
What is the correct way to backup and restore `~/.local.share/containers`?

When I try to do a backup and restore of `~/.local/share/containers` the restored data cannot be restarted due SELinux denials. If `setenforce 0` is set - the user containers will start.

Tried (and failed):

* with rsync

        # as user rsync fails with permission errors: as root
        rsync -avz containers/ containers.backup/
        mv containers containers.save
        mv containers.backup containers
        
        # now containers won't start with SELinux denials

* with tar

        # as user tar fails with permission errors: as root
        tar -cf - --xattrs --selinux  -C containers/ . | tar -C  containers.backup/ --selinux --xattrs -xf -
        mv containers containers.save
        mv containers.backup containers
        
        # now containers won't start with SELinux denials

**Comment by rhatdan** [comment]
Your Tar is not maintaining the XAttrs of the files, so they are loosing the fact that they were labeled container_file_t.
